dockerfiles/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3.7'
+
+services:
+  fuseki:
+    container_name: skosmos-fuseki
+    hostname: fuseki
+    image: stain/jena-fuseki
+    environment:
+      - ADMIN_PASSWORD=admin
+      - JVM_ARGS=-Xmx2g
+    ports:
+      - 9030:3030
+  # You can uncomment the line below to have a local volume bound onto the container, or
+  # visit https://hub.docker.com/r/stain/jena-fuseki/ for other alternatives
+  # volumes:
+  #  - ${PWD}/fuseki:/fuseki
+    volumes:
+      - type: bind
+        source: ./config/skosmos.ttl
+        target: /fuseki/configuration/skosmos.ttl
+  fuseki-cache:
+    container_name: skosmos-fuseki-cache
+    hostname: fuseki-cache
+    image: varnish
+    ports:
+      - 9031:80
+    volumes:
+      - type: bind
+        source: ./config/varnish-default.vcl
+        target: /etc/varnish/default.vcl
+  skosmos:
+    container_name: skosmos-web
+    hostname: skosmos
+    build:
+      context: .
+      dockerfile: Dockerfile.ubuntu
+    ports:
+      - 9090:80
+    depends_on:
+      - fuseki
+      - fuseki-cache
+    volumes:
+      - type: bind
+        source: ./config/config-docker-compose.ttl
+        target: /var/www/html/config.ttl
+
+# once the environment has been created load a vocabulary, e.g.:
+# >curl -I -X POST -H Content-Type:text/turtle -T yso-skos.ttl -G http://localhost:9030/skosmos/data --data-urlencode graph=http://www.yso.fi/onto/yso/
```

Comment:
It would be better if this matched STW and UNESCO...and perhaps even better to put this in the README file instead?

Reply:
Moved to README, now using STW and UNESCO.

Reply:
Done and done!